### ISSUE TITLE
Redundant Expect and Supertest require

### ISSUE THREAD

**Issue by cklanac** [JizongL/boilerplateForDev#1]
The expect and supertest requires on /test/app.spec.js are redundant since they are already loaded on the test/setup.js

```
const { expect } = require('chai');
const supertest = require('supertest');
```

**Comment by dsl2022** [comment]
Yes, I updated it afterwards, it was sent to you when I hadn't finished. they are declared globally so it's no need to do so. thanks!
